### PR TITLE
Update

### PR DESCRIPTION
**Pull request by luqaska**
- Imprace.com is not available anymore, so I removed it
- I added is-a.dev and FreeDNS to the list (Other Free Services)
- I added BearBlog (Dev Blogging Sites)
- I added Giscus and Gitalk (Commenting Platforms)
- Corrected some names

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1412,21 +1411,24 @@ Table of Contents
...
+  * [Dev.to](https://dev.to/) - Where programmers share ideas and help each other grow.
+  * [Hashnode](https://hashnode.com/) — Hassle-free Blogging Software for Developers!.
+  * [Medium](https://medium.com/) — Get smarter about what matters to you.
 
 **[⬆ back to top](#table-of-contents)**
 
 ## Commenting Platforms
   * [Staticman](https://staticman.net/) - Staticman is a Node.js application that receives user-generated content and uploads it as data files to a GitHub and/or GitLab repository, using Pull Requests.
+  * [Giscus](https://giscus.app/) - A lightweight comments widget built arround GitHub Discussions.
+  * [Gitalk](https://gitalk.github.io/) - A little comments widget built on GitHub Issues.
```
Comment (ripienaar) [inline README.md:1431]: hmm, bit of a thin line, but these are not really SaaS, they're effectively self hosted software.  We dont tend to list JS libraries without backend services.  And the service here that's free is github in these cases.

Comment (luqaska) [reply README.md:1431]: Ok, I'll remove it

Comment (luqaska) [reply README.md:1431]: Should I move it to the "Other Free Resources" section?

Comment (ripienaar) [reply README.md:1431]: No, I cant list every JS library, even if I just listed ones that talk to some kind of backend it would be thousands of lines.

Comment (luqaska) [reply README.md:1431]: Ok, I'll just remove it from the pull request then

**Comment by ripienaar** [comment]
sweet, thank you!

**Pull request merged**